src/user-picker/UserPickerCard.js

```diff
@@ -57,8 +61,9 @@ class UserPickerCard extends PureComponent<Props, State> {
     }
   };
 
-  handleUserPress = (email: string) => {
+  handleUserPress = (params: HandleUserPressParams) => {
```

Comment:
Like I said above, this `HandleUserPressParams` type is better off just written in line here.

But actually even better in this case: don't even write it at all! This is a good place for type inference. We can just write:
```
handleUserPress = ({ email }) => {
```
and because we have a nice type signature on `UserList`, Flow sees that the type of this method's parameter must be `{ email: string, fullName: string }`.

When doing that, it is essential that Flow really does succeed at the inference :smile: . Check that by hovering, or hitting C-k C-i to get the hover popup.